api_backend.yaml

```diff
@@ -0,0 +1,14 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: IO SPID commons
+  description: Common code for integrating SPID authentication.
+host: localhost
+paths: {}
+definitions:
+  EmailAddress:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions/v0.64.0/api/definitions.yaml#/EmailAddress"
+  FiscalCode:
```

Comment:
this doesn't need to be generated, can be imported from `italia-ts-commons/lib/strings`

Reply:
fixed in c5dd8fc